hack/make.ps1

```diff
@@ -84,13 +90,23 @@ param(
     [Parameter(Mandatory=$False)][switch]$PkgImports,
     [Parameter(Mandatory=$False)][switch]$GoFormat,
     [Parameter(Mandatory=$False)][switch]$TestUnit,
+    [Parameter(Mandatory=$False)][switch]$TestIntegration,
+    [Parameter(Mandatory=$False)][switch]$TestIntegrationCli,
     [Parameter(Mandatory=$False)][switch]$All
 )
 
 $ErrorActionPreference = "Stop"
 $ProgressPreference = "SilentlyContinue"
 $pushed=$False  # To restore the directory if we have temporarily pushed to one.
 
+# Work out the the -H parameter for the daemon under test (DASHH_DUT) and client under test (DASHH_CUT)
+#$DASHH_DUT="npipe:////./pipe/$COMMITHASH" # Can't do remote named pipe
```

Comment:
Is this debugging code, or kept here to be enabled at some point?

Reply:
Oh this changes are not ready, I should revert them :angel: